app/services/importers/okc_blotter/pdf.rb

```diff
@@ -113,8 +113,9 @@ def self.import_since_last_run
         (last_run_date + 1.day..DateTime.now.to_date).each do |date|
           perform(date)
         rescue StandardError => e
-          puts "error processing #{date}. Data not saved" # TODO: log these somewhere?
-          puts "error was: #{e}"
+          error = StandardError.new("Error processing #{date}. Data not saved. Message was: #{e}")
+          puts error
+          Raygun.track_exception(error)
```

Comment:
this prints the error and then sends it to raygun (rather than just printing it to console and getting no notifications)

Reply:
ok! creates a error object then adds it to what I assume is a list in Raygun.